docs/ref/models/querysets.txt

```diff
@@ -323,6 +323,12 @@ ones).
     If you intend to prefetch related objects from the alias,
     ``AttributeError`` will be raised.
 
+
+.. admonition:: :meth:`~.QuerySet.only()` is not yet supported
```

Comment:
"not supported"

the docs shouldn't imply it will come soon with "yet" as it might never happen